src/components/listbox/listbox.jsx

```diff
@@ -96,6 +74,29 @@ const Listbox = forwardRef(function Listbox(props, forwardedRef) {
     }
   }
 
+  useEffect(() => {
+    if (active) {
+      props.refs.forEach((optionRef, index) => {
+        if (optionRef.current) optionRef.current.setActive(index === activeIndex);
+      });
+    }
+  });
```

Comment:
```suggestion
  }, []);
```

Elsewhere, we are explicit about having no dependencies. We should probably standardize on that pattern.

Reply:
This implies only for the on mount lifecycle which we do not want in this case.

Reply:
You're right! In that case, we probably want:
```suggestion
  }, [active]);
```

Reply:
```suggestion
  }, [active, activeIndex]);
```

I think. Lemme see how that plays out.